File: bulk/java/ELInjection01.java
// CWE-94: EL Injection via Spring Expression Language
import org.springframework.expression.*;
import org.springframework.expression.spel.standard.*;
public class ELInjection01 {
    ExpressionParser parser = new SpelExpressionParser();
    public Object eval(String expr) {
        return parser.parseExpression(expr).getValue(); // arbitrary SpEL execution
    }
}

File: bulk/csharp/ELInjection01.cs
// CWE-94: Dynamic code execution via Roslyn CSharpScript
using Microsoft.CodeAnalysis.CSharp.Scripting;
public class ELInjection01 {
    public async Task<object> Eval(string code) {
        return await CSharpScript.EvaluateAsync(code); // arbitrary code execution
    }
}
